### ISSUE TITLE
The library needs a version

### ISSUE THREAD

**Issue by bast** [eloraiby/delaunay#10]
It would be good to give this code a version. This will make it easier for me to version the Python package accordingly.

**Comment by eloraiby** [comment]
Indeed, I will review your changes this weekend and add another option to the api to filter illegal input.

**Comment by bast** [comment]
Great. No rush. I am working on the Python interface. Looks good but want to test it more on my fork before contributing it.
